xknx/devices/remote_value_scaling.py

```diff
@@ -17,8 +17,8 @@ def __init__(self,
                  group_address_state=None,
                  device_name=None,
                  after_update_cb=None,
-                 range_from=100,
-                 range_to=0):
+                 range_from=0,
+                 range_to=100):
```

Comment:
this was the reason exposed percent values were inverted.

Reply:
nice catch, that really did hurt :/